css/qmwp-search-correlations.css

```diff
@@ -309,8 +309,16 @@
     align-items: center;
 }
 
+.details-and-controls > div:first-child {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Values of 0 shouldn't have units specified.](https://www.codacy.com/app/m_3/QuantiModo-WordPress-Plugin/file/1308809525/issues/source?bid=2836188&fileBranchId=2951770#l313)